packages/oidc-middleware/test/e2e/page-objects/HomePage.js

```diff
@@ -28,6 +28,12 @@ module.exports = class HomePage {
     await browser.wait(EC.urlIs(url), 50000);
   }
 
+  async performLogout() { 
+    const logoutButton = $('#logout');
+    await logoutButton.click();
+    await browser.wait(EC.not(EC.presenceOf(logoutButton)), 10000);
```

Comment:
Those are Vijet questions - the 50000 is notably large because that's the initial page load and bacon had issues with it, and most of the rest are 5000.  This got bumped when we were trying to figure out a core build issue, but I believe that ended up being something else, so I'll return this to 5000.

Reply:
@vijetmahabaleshwar-okta ^